data/src/codes.py

```diff
@@ -91,9 +91,15 @@ def _get_language_sizes(categories: List[str]) -> Dict[str, int]:
         ).json()
         for page in data["query"]["pages"].values():
             size = page["categoryinfo"]["size"]
-            language = re.search(
+
+            language_search = re.search(
                 r"Category:(.+?) terms with IPA pronunciation", page["title"]
-            ).group(1)
+            )
+
+            if not language_search:
+                continue
```

Comment:
I wasn't 100% sure what to do in the event that the regex was not satisfied, so I skip any pages where there's an issue

Reply:
Good question. Could you `logging.warning` it before `continue`?